pkg/url/kubernetes.go

```diff
@@ -0,0 +1,108 @@
+package url
+
+import (
+	"fmt"
+	"sort"
+
+	routev1 "github.com/openshift/api/route/v1"
+	componentlabels "github.com/openshift/odo/pkg/component/labels"
+	"github.com/openshift/odo/pkg/localConfigProvider"
+	"github.com/openshift/odo/pkg/occlient"
+	"github.com/pkg/errors"
+	"k8s.io/klog"
+)
+
+type kubernetesClient struct {
+	generic
+	isRouteSupported bool
+	client           occlient.Client
+}
+
+// ListCluster lists both route and ingress based URLs from the cluster
+func (k kubernetesClient) ListCluster() (URLList, error) {
```

Comment:
Should the function name be something else? It lists "URLs from the cluster" but the function name seems to indicate that it lists Cluster.

Reply:
Renamed to `ListFromCluster()`

Reply:
Thanks. That makes much more sense than the previous name. :+1: